### ISSUE TITLE
[Feature Request] Support for Jetbrains Fleet


### ISSUE THREAD

**Issue by shinayser** [flutter/flutter#94340]
Jetbrains just announced a new lightweight IDE with huge potential. 
We would love to see dart and flutter support for it (when the plugin's API gets available).
https://www.jetbrains.com/fleet/

**Comment by stuartmorgan** [comment]
What specifically is your request here? According to the linked site there is no plugin system at this point, so it's not clear what action you are looking for from the Flutter team.

**Comment by shinayser** [comment]
@stuartmorgan Not at the moment but I am opnening the issue for future tracking.

**Comment by jodinathan** [comment]
As Intellij seems to be a Google close partner I would expect Fleet to be Dart-ready as soon as it is launched. They seem to use LSP, however, I don't know if that is enough to use Dart/Flutter once if is open to public.  
We use WebStorm today and a lightweight IDE from Intellij is a very good news for us.

**Comment by bernaferrari** [comment]
lol what can they do? The thing is not even released.

**Comment by shinayser** [comment]
> lol what can they do? The thing is not even released.

https://github.com/flutter/flutter/issues/94340#issuecomment-981665425

**Comment by fwcd** [comment]
Considering that the Fleet website mentions language servers, integrating the Dart analysis server for code completion and similar IDE features should be pretty feasible.

**Comment by bernaferrari** [comment]
Intellij's Dart plugin is made by JetBrains, not Google. So it should come from them.

**Comment by v7lin** [comment]
any news update

**Comment by halildurmus** [comment]
There is an open issue for tracking adding support for _Dart_ in JetBrain's website: [Support for Dart : FL-10369](https://youtrack.jetbrains.com/issue/FL-10369/Support-for-Dart)

Please consider upvoting the issue so that it can be prioritized.

**Comment by kabirnayeem99** [comment]
Please think about adding support now.

**Comment by felipecastrosales** [comment]
news about this? looks interesting.

**Comment by followthemoney1** [comment]
+1

**Comment by mutlucan-iltir** [comment]
I would love to use fleet for my flutter projects it seem really nice right now.

**Comment by bernaferrari** [comment]
There is no plugin support on fleet yet.

**Comment by shtse8** [comment]
Is it possible to code in fleet and build from cmd?

**Comment by bernaferrari** [comment]
No, because there is no language support.

**Comment by berteodosio** [comment]
Do we have any info regarding contact between JetBrains and the Flutter team for support in the future?

**Comment by bernaferrari** [comment]
No, because jetbrains doesn't have plugin support yet. You need to wait for that.

**Comment by mariusvn** [comment]
For information, in their last blog post, at the end, the fleet team says
```
PS. Plugins support and plugins API is a work in progress.
```

**Comment by anderdobo** [comment]
The Flutter team at Google is not planning to build Flutter/Dart support for Jetbrains Fleet at this time. Community initiatives to do so are always welcome. Closing to reflect this.
